x-pack/plugins/infra/server/routes/metrics_explorer_views/README.md

```diff
@@ -77,38 +77,30 @@ Status code: 200
     "updatedAt": 1681398305034,
     "attributes": {
       "name": "Ad-hoc",
-      "isDefault": true,
```

Comment:
Are we not returning some of these anymore?

Reply:
The example in README.md was using the inventory view schema.

Reply:
I get them as a response when I get an inventory view and see them here: https://github.com/elastic/kibana/pull/160982/files#diff-0c4257ab44c5dd3e10f44c2742af7f54c76f2fa41cdb0a1d0e13d625bad1acd3R92

Reply:
These should always be present in the response objects. Fixed.